### PR TITLE
chore: update latest hyperswitch stable versions

### PR DESCRIPTION
**Pull request by jagan-jaya**

### CODE REVIEW

File: mac_deps.sh [hunk 1]
```diff
@@ -31,6 +31,7 @@ fi
 if ! command -v node &> /dev/null
 then
     brew install node
+    brew upgrade
```
Comment (nitesh-balla) [inline mac_deps.sh:34]: Can we do `brew upgrade` before `brew install node`?

File: lib/aws/eks.ts [hunk 2]
```diff
@@ -408,139 +408,167 @@ export class EksStack {
...
-
+    
+    const sdk_version = "0.27.2";
     const hypersChart = cluster.addHelmChart("HyperswitchServices", {
-      chart: "hyperswitch-helm",
-      repository: "https://juspay.github.io/hyperswitch-helm",
+      chart: "hyperswitch-stack",
+      repository: "https://juspay.github.io/hyperswitch-helm/v0.1.2",
       namespace: "hyperswitch",
       release: "hypers-v1",
```
Comment (nitesh-balla) [inline lib/aws/eks.ts:417]: Can we change the release name to something like `hyperswitch-0.1.2`? Would `hypers-v1 `suffice?

File: lib/aws/eks.ts [hunk 3]
```diff
@@ -408,139 +408,167 @@ export class EksStack {
...
-            serviceAccountAnnotations: {
-              "eks.amazonaws.com/role-arn": hyperswitchServiceAccountRole.roleArn,
+        "hyperswitch-app": {
+          loadBalancer: {
+            targetSecurityGroup: lbSecurityGroup.securityGroupId
+          },
+          services: {
+            router: {
+              version: "v1.107.0",
+              image: `juspaydotin/hyperswitch-router:v1.107.0-standalone`,
```
Comment (nitesh-balla) [inline lib/aws/eks.ts:431]: Can we change this to `juspaydotin/hyperswitch-router:v1.107.0`?

File: lib/aws/eks.ts [hunk 4]
```diff
@@ -408,139 +408,167 @@ export class EksStack {
...
-              kms_connector_onboarding_paypal_client_secret: kmsSecrets.kms_connector_onboarding_paypal_client_secret,
-              kms_connector_onboarding_paypal_partner_id: kmsSecrets.kms_connector_onboarding_paypal_partner_id,
-              kms_key_id: kms_key.keyId,
-              kms_key_region: kms_key.stack.region,
-              kms_encrypted_api_hash_key: kmsSecrets.kms_encrypted_api_hash_key,
-              admin_api_key: admin_api_key,
-              jwt_secret: "test_admin",
-              recon_admin_api_key: "test_admin",
+            producer: {
+              image: `juspaydotin/hyperswitch-producer:v1.107.0-standalone`
```
Comment (nitesh-balla) [inline lib/aws/eks.ts:435]: Same here.

File: lib/aws/eks.ts [hunk 5]
```diff
@@ -408,139 +408,167 @@ export class EksStack {
...
             },
-            master_enc_key: kmsSecrets.kms_encrypted_master_key,
-            locker: {
-              locker_readonly_key: locker ? locker.locker_ec2.locker_pair.public_key : "locker-key",
-              hyperswitch_private_key: locker ? locker.locker_ec2.tenant.private_key : "locker-key",
+            consumer: {
+              image: `juspaydotin/hyperswitch-consumer:v1.107.0-standalone`
```
Comment (nitesh-balla) [inline lib/aws/eks.ts:438]: Same here as well.

**Review by nitesh-balla — CHANGES_REQUESTED** [review]

File: lib/aws/eks.ts (hunk 5 above)
Comment (jagan-jaya) [reply lib/aws/eks.ts:438]: no  juspaydotin/hyperswitch-router:v1.107.0 will have email feature enabled which is not supported in cdk script for now, so we need to use standalone image and enable kms via env params

File: lib/aws/eks.ts (hunk 3 above)
Comment (nitesh-balla) [reply lib/aws/eks.ts:431]: Even in `juspaydotin/hyperswitch-router:v1.107.0`, we still have to enable kms via env params only. Can't we disable this email feature via envs?

**Review by nitesh-balla — APPROVED** [review]

**Pull request merged**